app/src/components/templateEditor/TemplateEditor.tsx

```diff
@@ -59,3 +60,30 @@ export const TemplateEditor = ({
     />
   );
 };
+
+export const TemplateEditorWrap = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  return (
+    <div
+      css={css`
+        & .cm-content {
+          padding: var(--ac-global-dimension-size-100)
+            var(--ac-global-dimension-size-250);
+        }
+        & .cm-gutter,
+        & .cm-content {
+          min-height: 75px;
```

Comment:
I kinda feel like we should only add the min-height if the editor is in non read-only mode. It's un-necessary spacing otherwise.

Reply:
Done